bindings/java/src/org/sleuthkit/datamodel/TimelineEventType.java

```diff
@@ -44,6 +44,8 @@
  number of levels and types. 
  */
 public interface TimelineEventType extends Comparable<TimelineEventType> {
+	
+	static final int EMAIL_FULL_DESCRIPTION_LENGTH_MAX = 150;
```

Comment:
This should be private.

Reply:
It cannot be.  Java doesn't appear to allow static private members of an interface.  I could just hard code the value (used at line 309).

Reply:
It's fine as it is, thanks. Learn something new everyday!